Source/PopupView.swift

```diff
@@ -303,6 +306,7 @@ public struct Popup<PopupContent: View>: ViewModifier {
     // MARK: - Public Properties
 
     var position: Position
+    var type: PopupType
```

Comment:
1 position
2 type
please take care of parameters order - it should be the same in declaration, init, etc. type looks like the most important one so it should be on top